README.md

```diff
@@ -1,57 +1,11 @@
-# Northcoders News API
+You will need to create two .env files for your project: .env.test and
+.env.development. Into each, add PGDATABASE=<database_name_here>, with the
```

Comment:
I would specify what the databases are called, and which one needs to go into each. 
Getting them to hunt around for the database names will increase the chance of them doing it wrong